client/src/components/users/manageUsers/UsersActions.js

```diff
@@ -82,28 +86,27 @@ export const updateUserSuccess = data => ({
   data
 });
 
-export const getUserSuccess = () => ({
-  type: types.GET_USER_SUCCESS
+export const getUserSuccess = data => ({
+  type: types.GET_USER_SUCCESS,
+  data
 });
 
 export const updateFailureMessage = errorMessage => ({
   type: types.UPDATE_FAILURE_MESSAGE,
   errorMessage
 });
 
-export const updateUserProfile = updatedUser => (dispatch) => {
-  return axios.put(`/users/${updatedUser.id}`, updatedUser)
-    .then((response) => {
+export const updateUserProfile = updatedUser => (dispatch) => axios.put(`/users/${updatedUser.id}`, updatedUser)
```

Comment:
It's great practice to omit the braces If your function takes a single argument.